monitoring/generator.go

```diff
@@ -185,7 +185,7 @@ type Observable struct {
 	//
 	// See README.md for why it is intentionally impossible to create a dashboard to monitor
 	// something without at least a warning alert being defined.
```

Comment:
```suggestion
	// It is possible to create a dashboard to monitor something without any alerts defined,
	// but we strongly recommend that you provide at least a warning alert where possible.
	// 
	// See https://about.sourcegraph.com/handbook/engineering/observability/monitoring_pillars
	// to learn more.
```

(we can update the page later, but I think we can still base a strong recommendation for alerts on the current (though now outdated) reasoning)

Reply:
We don't, though! Often times we don't even know what "good" or "bad" looks like, so we should not make engineers worry about that for every panel. Sometimes we don't even care about alerting on something, even if it's a warning. To be honest, I find warning alerts just noise — no one looks at them and they aren't actionable.

Reply:
I'll update the handbook after this and fix this comment then.

Reply:
> so we should not make engineers worry about that for every pane

I agree we should not enforce it, as it was discussed, but as we talked about when we paired, there should be an explicit "no alert" if no alert is needed, and not a pass by default, as it both documents that we intentionally decided not to put an alert (and we didnt just forget) and encourages you to think about adding one.

> To be honest, I find warning alerts just noise — no one looks at them and they aren't actionable.

I dont think this is accurate, warnings should be actioned and reviewed, they are not paged because some of them are not worth waking up at 4am to fix.
As in the example I provided before, an pod OOMing every 1h but coming back healthy, its not something we need to fix at 4AM, but its something we need to fix.

Reply:
I thought this was pretty insightful into why the generator was built to discourage no alerts: https://about.sourcegraph.com/handbook/engineering/observability/monitoring_pillars#faq-i-have-a-purely-information-metric-i-don-t-want-to-define-an-alert-for-it-how-do-i-do-that

While I agree that they are alerts are not always useful on every graph, it would be great if we could update our direction first (and maybe include examples those cases you have experience with where certain types of information cannot have useful alerts for reference) before we continue work on the generator! I went ahead and opened https://github.com/sourcegraph/about/pull/2000 , would appreciate if you could let us know what specific changes you think we should make :)

Thanks for driving these improvements @tsenart !